### PR TITLE
Share Button Added

### PR DESCRIPTION
**Pull request by yash37158**
Share Button added successfully , Now share your this app on your favourite social media platforms

### CODE REVIEW

**Comment by yash37158** [comment]
how should i resolve this!

**Comment by abhinav12k** [comment]
@yash37158, Please fix the codacy issues and attach a screenshot of the share button.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 <resources>
     <string name="app_name">MoviFreak</string>
-    <string name="api_key">enter_your_api</string>
+    <string name="api_key">262dee09558f1b7f353146f9570b3850</string>
```
Comment (abhinav12k) [inline app/src/main/res/values/strings.xml:3]: Remove the api key from here

**Comment by yash37158** [comment]
okay

**Comment by yash37158** [comment]
so after removing i can create a pull request and should i checkout from
feature  branch ??

**Pull request closed without merging**